package.json

```diff
@@ -6,65 +6,67 @@
-    "build": "webpack -p --env prod --display-error-details --config webpack.config.js",
-    "dev": "webpack --env dev --progress --colors --watch --config webpack.config.js",
-    "lint": "./node_modules/.bin/eslint designsafe/static/scripts/**/*.js",
-    "report-coverage": "./node_modules/.bin/codecov"
+    "report-coverage": "./node_modules/.bin/codecov",
+    "lint": "eslint designsafe/static/scripts/**/*.js",
+    "build": "NODE_ENV=production BABEL_ENV=production webpack --env production --config webpack.config.js",
+    "dev": "NODE_ENV=development BABEL_ENV=development webpack --env development --watch --color --config webpack.config.js"
   },
-  "author": "Matthew R Hanlon <mrhanlon@tacc.utexas.edu>",
-  "contributors": [
-    "Matthew R Hanlon <mrhanlon@tacc.utexas.edu>",
-    "Steve Mock <mock@tacc.utexas.edu>",
-    "Josue Balandrano Coronel <jcoronel@tacc.utexas.edu>",
-    "Manuel Rojas <mrojas@tacc.utexas.edu>",
-    "Marjo Poindexter <marjo@tacc.utexas.edu>",
-    "Hedda Prochaska <hedda@tacc.utexas.edu>",
-    "Alex Rocha <rochaa@tacc.utexas.edu>"
-  ],
   "license": "MIT",
   "devDependencies": {
     "@babel/core": "^7.10.3",
     "@babel/preset-env": "^7.10.3",
+    "babel-eslint": "^10.0.3",
     "babel-loader": "^8.1.0",
-    "css-loader": "^0.28.11",
+    "babel-plugin-angularjs-annotate": "^0.9.0",
+    "clean-webpack-plugin": "^3.0.0",
+    "css-loader": "^5.0.0",
     "elasticdump": "^3.3.7",
-    "extract-text-webpack-plugin": "^3.0.2",
-    "file-loader": "^2.0.0",
+    "eslint": "^7.2.0",
+    "eslint-config-airbnb-base": "^14.2.0",
+    "eslint-config-prettier": "^6.7.0",
+    "eslint-config-standard": "^14.1.0",
+    "eslint-import-resolver-webpack": "^0.12.1",
+    "eslint-loader": "^3.0.3",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-node": "^11.0.0",
+    "eslint-plugin-prettier": "^3.1.1",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-standard": "^4.0.1",
+    "file-loader": "^5.0.2",
     "html-loader": "^0.5.5",
-    "html-webpack-plugin": "^2.30.1",
-    "istanbul-instrumenter-loader": "^3.0.1",
     "jasmine": "^3.3.0",
     "jasmine-core": "^3.2.1",
-    "karma": "^3.1.1",
+    "karma": "^5.2.3",
     "karma-chrome-launcher": "^2.2.0",
-    "karma-coverage": "^1.1.2",
+    "karma-coverage": "^2.0.3",
     "karma-firefox-launcher": "^0.1.7",
     "karma-jasmine": "^1.1.2",
     "karma-jasmine-html-reporter": "^0.1.8",
     "karma-sourcemap-loader": "^0.3.7",
     "karma-spec-reporter": "0.0.32",
-    "ng-annotate-webpack-plugin": "^0.2.1-pre",
+    "karma-webpack": "^4.0.2",
+    "mini-css-extract-plugin": "^1.1.1",
     "ngtemplate-loader": "^2.0.1",
     "node-sass": "^4.14.1",
-    "sass-loader": "^6.0.7",
-    "style-loader": "^0.20.3",
-    "webpack": "^3.12.0",
-    "webpack-livereload-plugin": "^0.10.0"
+    "prettier": "^2.1.2",
+    "sass-loader": "^8.0.0",
+    "style-loader": "^1.0.1",
+    "webpack": "^5.1.3",
```

Comment:
`karma-webpack` doesn't work with webpack 5 (and webpack 4 causes the test runner to throw really cryptic dependency injection errors). I think as long as we're using Angular 1 we're stick with Webpack 3.

Reply:
Yeah that's a bummer. Seemed like others had been able to get it working with webpack 5, but might have to roll it back if I can't

Reply:
Got it working with v4